.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2
+jobs:
+  build_docs:
+    docker:
+      - image: circleci/python:3.6-stretch
+    steps:
+      # Get our data and merge with upstream
+      - run: sudo apt-get update
+      - checkout
+      # Python env
+      - run: echo "export PATH=~/.local/bin:$PATH" >> $BASH_ENV
+
+      - restore_cache:
+          keys:
+            - cache-pip
+      # Install packages needed for sphinx build
+      - run: pip install --user alabaster traitlets tornado sphinx recommonmark sphinx_copybutton escapism kubernetes prometheus_client jupyterhub docker python-json-logger
+      - run: pip install -e git+git://github.com/jupyterhub/alabaster_jupyterhub.git#egg=master
```

Comment:
Not sure why this fails, I'd try `pip install https://github.com/jupyterhub/alabaster-jupyterhub/archive/master.zip` and see what happens

Reply:
Added some modifications to the theme https://github.com/jupyterhub/alabaster-jupyterhub/pull/3